src/view/shell/createNativeStackNavigatorWithAuth.tsx

```diff
@@ -36,7 +36,7 @@ import {Onboarding} from '#/screens/Onboarding'
 import {SignupQueued} from '#/screens/SignupQueued'
 import {atoms as a} from '#/alf'
 import {BottomBarWeb} from './bottom-bar/BottomBarWeb'
-import {DesktopLeftNav} from './desktop/LeftNav'
+import DesktopLeftNav from './desktop/LeftNav'
```

Comment:
Let's put `DesktopLeftNav` back to a named export, just so this file doesn't need to be changed